optimum/exporters/onnx/__main__.py

```diff
@@ -130,13 +126,20 @@ def main():
         )
 
     if model.config.is_encoder_decoder and args.for_ort:
-        onnx_inputs, onnx_outputs = export_encoder_decoder_model(
-            model,
-            onnx_config,
-            args.opset,
-            args.output.parent.joinpath("encoder_model.onnx"),
-            args.output.parent.joinpath("decoder_model.onnx"),
-            args.output.parent.joinpath("decoder_with_past_model.onnx"),
+        onnx_inputs, onnx_outputs = export_models(
+            model=model,
+            onnx_config=onnx_config,
+            opset=args.opset,
+            output_dir=args.output.parent,
+            fn_get_models_from_config=get_encoder_decoder_models_for_export,
+        )
+    elif args.for_ort and task.startswith("causal-lm"):
+        onnx_inputs, onnx_outputs = export_models(
+            model=model,
+            onnx_config=onnx_config,
+            opset=args.opset,
+            output_dir=args.output.parent,
+            fn_get_models_from_config=get_decoder_models_for_export,
```

Comment:
That's a small nit, but I think we can just do:
```python
if args.ort and (model.config.is_encoder_decoder or task.startswith("causal-lm"):
    fn_get_models_from_configs = get_encoder_decoder_model if model.config.is_encoder_decoder else get ....
    onnx_inputs, onnx_outputs = export (....)
```

Reply:
yes good catch, thanks!